lib/beaker/shared/options_resolver.rb

```diff
@@ -0,0 +1,21 @@
+module Beaker
+  module Shared
+    # Methods for parsing options.
+    # - determine if a mode should be run in parallel
+    module OptionsResolver
+      def run_in_parallel?(opts, options, mode)
```

Comment:
@puppetlabs/beaker did we ever come up with an agreement of what we really mean in the code base when we name variables `opts` or `options`? If I remember correctly, `opts` are local options, and `options` refer to the global options map we pass around. Or, is that even too limited of a definition?